CHANGELOG.md

```diff
@@ -86,6 +87,7 @@ Querying with using `{__mimir_storage__="ephemeral"}` selector no longer works.
 
 ### Mimirtool
 
+* [CHANGE] Blocks uploaded with the `mimirtool backfill` command are now by default verified by a compactor before the upload is finalized. #3411
```

Comment:
I don't think we need any entry for Mimirtool. It's only a change in Mimir.

I will merge as-is, but if you agree, please send a PR to remove this entry.

Reply:
Agreed, https://github.com/grafana/mimir/pull/4369